source/ranges.tex

```diff
@@ -194,6 +194,14 @@
   template<class T>
     inline constexpr bool enable_borrowed_range<ref_view<T>> = true;
 
+  // \ref{range.owning.view}, owning view
+  template<range R>
+    requires @\seebelow@
```

Comment:
Issue with spec (note: doesn't apply here): We're inconsistent about wether to follow "requires @\seebelow@" with a ";" when declaring functions.  I think we should always end a requires clause with a ";" for syntactic reasons.

Reply:
Yes, I should have specified that I was talking about cases where @\seebelow@ was used as the trailing requires-clause. I thought I saw cases where the ";" was missing (and added this comment as a reminder to deal with them), but now I can't find any (tired eyes playing tricks on me?).